pyproject.toml

```diff
@@ -13,7 +13,7 @@ authors = [
 requires-python = ">=3.8"
 dependencies = [
     "click>=6",
-    "grimp>=3.1",
+    "git+https://github.com/seddonym/grimp.git@88d49122dd66fe6d092a69a5e8a302e667e908d1",
```

Comment:
This commit needs to be removed before merge! ⚠️ 

I've tested locally against this version of grimp, all seems to work well.

Reply:
or more accurately, the new version of grimp needs to be published and set here.